LEDMatrix.py

```diff
@@ -0,0 +1,5 @@
+class LEDMatrix(list):
+    def __init__(self, microcontroller):
```

Comment:
[please add a blank line between class definition and method](https://www.python.org/dev/peps/pep-0008/#blank-lines)